tests/integration/kubernetes/gha-run.sh

```diff
@@ -345,6 +345,54 @@ function run_tests() {
 	popd
 }
 
+# Print a report about tests executed.
+#
+# Crawl over the output files found on each "reports/yyyy-mm-dd-hh:mm:ss"
+# directory.
+#
+function report_tests() {
+	local reports_dir="${kubernetes_dir}/reports"
+	local reports
+	local ok
+	local not_ok
+	local flaky
+	local status
+
+	if [ ! -d "$reports_dir" ]; then
+		info "no reports directory found: $reports_dir"
+		return
+	fi
+
+	reports=$(ls "$reports_dir")
+	for report in $reports; do
+		ok=($(find "${reports_dir}/${report}" -name "ok-*.out"))
+		not_ok=($(find "${reports_dir}/${report}" -name "not_ok-*.out"))
+		flaky=($(find "${reports_dir}/${report}" -name "flaky-*.out"))
+
+		cat <<-EOF
+		SUMMARY ($(basename "$report")):
```

Comment:
Originally I wanted to suggest moving this section at the end, but since we need to unwrap this in gh output this way it's better visible.

On the other hand if we reported non-zero status in case of failed cases, the section would be unwrapped and by scrolling to he bottom we should see exactly what went wrong. What do you think?

Reply:
Another pro for failing the case would be better visibility because as a newbie looking at my failed ci I'd have to go through the test results, but if this step failed as well I'd have seen the summary, which should help me significantly.